point_cloud_transport/include/point_cloud_transport/publisher_plugin.hpp

```diff
@@ -147,6 +161,16 @@ class PublisherPlugin
     const std::string & base_topic,
     rmw_qos_profile_t custom_qos = rmw_qos_profile_default,
     const rclcpp::PublisherOptions & options = rclcpp::PublisherOptions()) = 0;
```

Comment:
Should we provide implementation for old deprecated pure virtual functions in `publisher_plugin` and `subscriber_plugin` ?
```suggestion
    const rclcpp::PublisherOptions & options = rclcpp::PublisherOptions())
  {
    advertiseImpl(
      node_interfaces, base_topic,
      rclcpp::QoS(rclcpp::QoSInitialization::from_rmw(custom_qos), custom_qos),
      options);
  }
```